src/benchmarklib/benchmark_utils.hpp

```diff
@@ -24,6 +25,8 @@ using TimePoint = std::chrono::high_resolution_clock::time_point;
 using NamedQuery = std::pair<std::string, std::string>;
 using NamedQueries = std::vector<NamedQuery>;
 
+using EncodingMapping = std::unordered_map<std::string, std::map<std::string, ColumnEncodingSpec>>;
```

Comment:
This maps table name -> column name -> encoding spec, right? I'd love to have a name here that tells me that.
`EncodingSpecByTableAndColumn` would get that across, but be a horrid name...

Reply:
i'll try to come up with something better :)